<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1652981778977' stop='1652981813766'>
    <name>Access Admin Panel</name>
    <title>Access Admin Panel</title>
    <test-cases>
        <test-case start='1652981778977' status='broken' stop='1652981813748'>
            <name>Login</name>
            <title>Login</title>
            <labels>
                <label name='language' value='javascript'/>
                <label name='framework' value='wdio'/>
                <label name='thread' value='0-0'/>
            </labels>
            <parameters>
                <parameter kind='argument' name='device' value='TestesEbacs 12.0-12'/>
            </parameters>
            <steps/>
            <attachments>
                <attachment title='Execution video' source='ea7338df-446e-46f6-b52e-41f8889d31db-attachment.mp4' type='video/mp4' size='191'/>
            </attachments>
            <failure>
                <message>element ("id:login_enter_password") still not existing after 20000ms</message>
                <stack-trace>Error: element ("id:login_enter_password") still not existing after 20000ms
                    at C:\Users\Gabriel Roquim QA\OneDrive\EBAC\Módulo 17 - Testes Mobile - Parte 2\MobileShop\A17-Exercicio\node_modules\webdriverio\build\commands\browser\waitUntil.js:66:23
                    at async Element.wrapCommandFn (C:\Users\Gabriel Roquim QA\OneDrive\EBAC\Módulo 17 - Testes Mobile - Parte 2\MobileShop\A17-Exercicio\node_modules\@wdio\utils\build\shim.js:137:29)
                    at async Element.elementErrorHandlerCallbackFn (C:\Users\Gabriel Roquim QA\OneDrive\EBAC\Módulo 17 - Testes Mobile - Parte 2\MobileShop\A17-Exercicio\node_modules\webdriverio\build\middlewares.js:24:32)
                    at async Element.wrapCommandFn (C:\Users\Gabriel Roquim QA\OneDrive\EBAC\Módulo 17 - Testes Mobile - Parte 2\MobileShop\A17-Exercicio\node_modules\@wdio\utils\build\shim.js:137:29)
                    at async Element.wrapCommandFn (C:\Users\Gabriel Roquim QA\OneDrive\EBAC\Módulo 17 - Testes Mobile - Parte 2\MobileShop\A17-Exercicio\node_modules\@wdio\utils\build\shim.js:137:29)
                    at async Element.elementErrorHandlerCallbackFn (C:\Users\Gabriel Roquim QA\OneDrive\EBAC\Módulo 17 - Testes Mobile - Parte 2\MobileShop\A17-Exercicio\node_modules\webdriverio\build\middlewares.js:24:32)
                    at async Element.wrapCommandFn (C:\Users\Gabriel Roquim QA\OneDrive\EBAC\Módulo 17 - Testes Mobile - Parte 2\MobileShop\A17-Exercicio\node_modules\@wdio\utils\build\shim.js:137:29)
                    at async loginScreen.goToLogin (C:\Users\Gabriel Roquim QA\OneDrive\EBAC\Módulo 17 - Testes Mobile - Parte 2\MobileShop\a17-exercicio\test\screens\login.screen.js:25:9)
                    at async Context.&lt;anonymous> (C:\Users\Gabriel Roquim QA\OneDrive\EBAC\Módulo 17 - Testes Mobile - Parte 2\MobileShop\a17-exercicio\test\specs\cadastroProduto.spec.js:20:9)</stack-trace>
            </failure>
        </test-case>
    </test-cases>
</ns2:test-suite>